### ISSUE TITLE
Problem: Cannot start minikube with hyperv driver in Windows 10

### ISSUE THREAD

**Issue by reza-ebrahimi** [kubernetes/minikube#1523]
I have used following command and its doesn't work:

`$ cat ~/.minikube/machines/minikube/config.json | grep DriverName`
`"DriverName": "virtualbox",`
`minikube.exe start --vm-driver="hyperv" --hyperv-virtual-switch="Primary Virtual Switch"`

But I have found another problem, if you set `--vm-driver hyperv` argument to `minikube`, it doesnot override` config.json` file value if it is `virtualbox`, and still assume `virtualbox` driver.

and also I have to change `DriverName`  in `config.json` file to `hyperv` in path `.minikube\machines\minikube`, and `minikube` give me following error:

Platform: `Windows 10`
Please not that `hyper-v` feature is enabled in my system and docker itself works fine. 

![1](https://cloud.githubusercontent.com/assets/11276982/26443498/e339b812-414d-11e7-9016-beae84ea3382.png)
![3](https://cloud.githubusercontent.com/assets/11276982/26443547/1b394e76-414e-11e7-9439-d59891c42d38.png)
![2](https://cloud.githubusercontent.com/assets/11276982/26443555/236f9d52-414e-11e7-9931-ac9642218b5a.png)

![4](https://cloud.githubusercontent.com/assets/11276982/26443561/30d40a5a-414e-11e7-8346-aa478c301d7a.png)

And log is:

`PS C:\kubernetes> $env:HYPERV_VIRTUAL_SWITCH="Primary Virtual Switch"`
`PS C:\kubernetes> ./minikube.exe start --vm-driver hyperv --hyperv-virtual-switch="Primary Virtual Switch" --v=9`

>Starting local Kubernetes v1.6.0 cluster...
Starting VM...
Found binary path at C:\kubernetes\minikube.exe
Launching plugin server for driver virtualbox
Plugin server listening at address 127.0.0.1:2996
() Calling .GetVersion
Using API Version  1
() Calling .SetConfigRaw
() Calling .GetMachineName
(minikube) Calling .GetState
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo minikube --machinereadable
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | name="minikube"
(minikube) DBG | groups="/"
(minikube) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)"
(minikube) DBG | UUID="66e150a4-62a4-4502-9b73-88880f30cc8a"
(minikube) DBG | CfgFile="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\minikube.vbox"
(minikube) DBG | SnapFldr="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\Snapshots"
(minikube) DBG | LogFldr="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\Logs"
(minikube) DBG | hardwareuuid="66e150a4-62a4-4502-9b73-88880f30cc8a"
(minikube) DBG | memory=2048
(minikube) DBG | pagefusion="off"
(minikube) DBG | vram=8
(minikube) DBG | cpuexecutioncap=100
(minikube) DBG | hpet="on"
(minikube) DBG | chipset="piix3"
(minikube) DBG | firmware="BIOS"
(minikube) DBG | cpus=2
(minikube) DBG | pae="on"
(minikube) DBG | longmode="on"
(minikube) Calling .Start
(minikube) DBG | triplefaultreset="off"
(minikube) DBG | apic="on"
(minikube) DBG | x2apic="off"
(minikube) DBG | cpuid-portability-level=0
(minikube) DBG | bootmenu="disabled"
(minikube) DBG | boot1="dvd"
(minikube) DBG | boot2="dvd"
(minikube) DBG | boot3="disk"
(minikube) DBG | boot4="none"
(minikube) DBG | acpi="on"
(minikube) DBG | ioapic="on"
(minikube) DBG | biosapic="apic"
(minikube) DBG | biossystemtimeoffset=0
(minikube) DBG | rtcuseutc="on"
(minikube) DBG | hwvirtex="on"
(minikube) DBG | nestedpaging="on"
(minikube) DBG | largepages="on"
(minikube) DBG | vtxvpid="on"
(minikube) DBG | vtxux="on"
(minikube) DBG | paravirtprovider="default"
(minikube) DBG | effparavirtprovider="kvm"
(minikube) DBG | VMState="poweroff"
(minikube) DBG | VMStateChangeTime="2017-05-25T09:57:16.000000000"
(minikube) DBG | monitorcount=1
(minikube) DBG | accelerate3d="off"
(minikube) DBG | accelerate2dvideo="off"
(minikube) DBG | teleporterenabled="off"
(minikube) DBG | teleporterport=0
(minikube) DBG | teleporteraddress=""
(minikube) DBG | teleporterpassword=""
(minikube) DBG | tracing-enabled="off"
(minikube) DBG | tracing-allow-vm-access="off"
(minikube) DBG | tracing-config=""
(minikube) DBG | autostart-enabled="off"
(minikube) DBG | autostart-delay=0
(minikube) DBG | defaultfrontend=""
(minikube) DBG | storagecontrollername0="SATA"
(minikube) DBG | storagecontrollertype0="IntelAhci"
(minikube) DBG | storagecontrollerinstance0="0"
(minikube) DBG | storagecontrollermaxportcount0="30"
(minikube) DBG | storagecontrollerportcount0="30"
(minikube) DBG | storagecontrollerbootable0="on"
(minikube) DBG | "SATA-0-0"="C:\Users\asus\.minikube\machines\minikube\boot2docker.iso"
(minikube) DBG | "SATA-ImageUUID-0-0"="31d1753b-8b2c-426b-a3f5-3630e1febd1e"
(minikube) DBG | "SATA-tempeject"="off"
(minikube) DBG | "SATA-IsEjected"="off"
(minikube) DBG | "SATA-1-0"="C:\Users\asus\.minikube\machines\minikube\disk.vmdk"
(minikube) DBG | "SATA-ImageUUID-1-0"="5746918b-9768-4c3c-8441-fc4c7f7bb2c3"
(minikube) DBG | "SATA-2-0"="none"
(minikube) DBG | "SATA-3-0"="none"
(minikube) DBG | "SATA-4-0"="none"
(minikube) DBG | "SATA-5-0"="none"
(minikube) DBG | "SATA-6-0"="none"
(minikube) DBG | "SATA-7-0"="none"
(minikube) DBG | "SATA-8-0"="none"
(minikube) DBG | "SATA-9-0"="none"
(minikube) DBG | "SATA-10-0"="none"
(minikube) DBG | "SATA-11-0"="none"
(minikube) DBG | "SATA-12-0"="none"
(minikube) DBG | "SATA-13-0"="none"
(minikube) DBG | "SATA-14-0"="none"
(minikube) DBG | "SATA-15-0"="none"
(minikube) DBG | "SATA-16-0"="none"
(minikube) DBG | "SATA-17-0"="none"
(minikube) DBG | "SATA-18-0"="none"
(minikube) DBG | "SATA-19-0"="none"
(minikube) DBG | "SATA-20-0"="none"
(minikube) DBG | "SATA-21-0"="none"
(minikube) DBG | "SATA-22-0"="none"
(minikube) DBG | "SATA-23-0"="none"
(minikube) DBG | "SATA-24-0"="none"
(minikube) DBG | "SATA-25-0"="none"
(minikube) DBG | "SATA-26-0"="none"
(minikube) DBG | "SATA-27-0"="none"
(minikube) DBG | "SATA-28-0"="none"
(minikube) DBG | "SATA-29-0"="none"
(minikube) DBG | natnet1="nat"
(minikube) DBG | macaddress1="080027F5725A"
(minikube) DBG | cableconnected1="on"
(minikube) DBG | nic1="nat"
(minikube) DBG | nictype1="82540EM"
(minikube) DBG | nicspeed1="0"
(minikube) DBG | mtu="0"
(minikube) DBG | sockSnd="64"
(minikube) DBG | sockRcv="64"
(minikube) DBG | tcpWndSnd="64"
(minikube) DBG | tcpWndRcv="64"
(minikube) Check network to re-create if needed...
(minikube) DBG | Forwarding(0)="ssh,tcp,127.0.0.1,1948,,22"
(minikube) DBG | hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter #2"
(minikube) DBG | macaddress2="0800276B6570"
(minikube) DBG | cableconnected2="on"
(minikube) DBG | nic2="hostonly"
(minikube) DBG | nictype2="82540EM"
(minikube) DBG | nicspeed2="0"
(minikube) DBG | nic3="none"
(minikube) DBG | nic4="none"
(minikube) DBG | nic5="none"
(minikube) DBG | nic6="none"
(minikube) DBG | nic7="none"
(minikube) DBG | nic8="none"
(minikube) DBG | hidpointing="ps2mouse"
(minikube) DBG | hidkeyboard="ps2kbd"
(minikube) DBG | uart1="off"
(minikube) DBG | uart2="off"
(minikube) DBG | uart3="off"
(minikube) DBG | uart4="off"
(minikube) DBG | lpt1="off"
(minikube) DBG | lpt2="off"
(minikube) DBG | audio="dsound"
(minikube) DBG | clipboard="disabled"
(minikube) DBG | draganddrop="disabled"
(minikube) DBG | vrde="off"
(minikube) DBG | usb="off"
(minikube) DBG | ehci="off"
(minikube) DBG | xhci="off"
(minikube) DBG | SharedFolderNameMachineMapping1="c/Users"
(minikube) DBG | SharedFolderPathMachineMapping1="\\?\c:\Users"
(minikube) DBG | vcpenabled="off"
(minikube) DBG | vcpscreens=0
(minikube) DBG | vcpfile="C:\Users\asus\.minikube\machines\minikube\minikube\minikube.webm"
(minikube) DBG | vcpwidth=1024
(minikube) DBG | vcpheight=768
(minikube) DBG | vcprate=512
(minikube) DBG | vcpfps=25
(minikube) DBG | GuestMemoryBalloon=0
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe showvminfo minikube --machinereadable
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | name="minikube"
(minikube) DBG | groups="/"
(minikube) DBG | ostype="Linux 2.6 / 3.x / 4.x (64-bit)"
(minikube) DBG | UUID="66e150a4-62a4-4502-9b73-88880f30cc8a"
(minikube) DBG | CfgFile="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\minikube.vbox"
(minikube) DBG | SnapFldr="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\Snapshots"
(minikube) DBG | LogFldr="C:\\Users\\asus\\.minikube\\machines\\minikube\\minikube\\Logs"
(minikube) DBG | hardwareuuid="66e150a4-62a4-4502-9b73-88880f30cc8a"
(minikube) DBG | memory=2048
(minikube) DBG | pagefusion="off"
(minikube) DBG | vram=8
(minikube) DBG | cpuexecutioncap=100
(minikube) DBG | hpet="on"
(minikube) DBG | chipset="piix3"
(minikube) DBG | firmware="BIOS"
(minikube) DBG | cpus=2
(minikube) DBG | pae="on"
(minikube) DBG | longmode="on"
(minikube) DBG | triplefaultreset="off"
(minikube) DBG | apic="on"
(minikube) DBG | x2apic="off"
(minikube) DBG | cpuid-portability-level=0
(minikube) DBG | bootmenu="disabled"
(minikube) DBG | boot1="dvd"
(minikube) DBG | boot2="dvd"
(minikube) DBG | boot3="disk"
(minikube) DBG | boot4="none"
(minikube) DBG | acpi="on"
(minikube) DBG | ioapic="on"
(minikube) DBG | biosapic="apic"
(minikube) DBG | biossystemtimeoffset=0
(minikube) DBG | rtcuseutc="on"
(minikube) DBG | hwvirtex="on"
(minikube) DBG | nestedpaging="on"
(minikube) DBG | largepages="on"
(minikube) DBG | vtxvpid="on"
(minikube) DBG | vtxux="on"
(minikube) DBG | paravirtprovider="default"
(minikube) DBG | effparavirtprovider="kvm"
(minikube) DBG | VMState="poweroff"
(minikube) DBG | VMStateChangeTime="2017-05-25T09:57:16.000000000"
(minikube) DBG | monitorcount=1
(minikube) DBG | accelerate3d="off"
(minikube) DBG | accelerate2dvideo="off"
(minikube) DBG | teleporterenabled="off"
(minikube) DBG | teleporterport=0
(minikube) DBG | teleporteraddress=""
(minikube) DBG | teleporterpassword=""
(minikube) DBG | tracing-enabled="off"
(minikube) DBG | tracing-allow-vm-access="off"
(minikube) DBG | tracing-config=""
(minikube) DBG | autostart-enabled="off"
(minikube) DBG | autostart-delay=0
(minikube) DBG | defaultfrontend=""
(minikube) DBG | storagecontrollername0="SATA"
(minikube) DBG | storagecontrollertype0="IntelAhci"
(minikube) DBG | storagecontrollerinstance0="0"
(minikube) DBG | storagecontrollermaxportcount0="30"
(minikube) DBG | storagecontrollerportcount0="30"
(minikube) DBG | storagecontrollerbootable0="on"
(minikube) DBG | "SATA-0-0"="C:\Users\asus\.minikube\machines\minikube\boot2docker.iso"
(minikube) DBG | "SATA-ImageUUID-0-0"="31d1753b-8b2c-426b-a3f5-3630e1febd1e"
(minikube) DBG | "SATA-tempeject"="off"
(minikube) DBG | "SATA-IsEjected"="off"
(minikube) DBG | "SATA-1-0"="C:\Users\asus\.minikube\machines\minikube\disk.vmdk"
(minikube) DBG | "SATA-ImageUUID-1-0"="5746918b-9768-4c3c-8441-fc4c7f7bb2c3"
(minikube) DBG | "SATA-2-0"="none"
(minikube) DBG | "SATA-3-0"="none"
(minikube) DBG | "SATA-4-0"="none"
(minikube) DBG | "SATA-5-0"="none"
(minikube) DBG | "SATA-6-0"="none"
(minikube) DBG | "SATA-7-0"="none"
(minikube) DBG | "SATA-8-0"="none"
(minikube) DBG | "SATA-9-0"="none"
(minikube) DBG | "SATA-10-0"="none"
(minikube) DBG | "SATA-11-0"="none"
(minikube) DBG | "SATA-12-0"="none"
(minikube) DBG | "SATA-13-0"="none"
(minikube) DBG | "SATA-14-0"="none"
(minikube) DBG | "SATA-15-0"="none"
(minikube) DBG | "SATA-16-0"="none"
(minikube) DBG | "SATA-17-0"="none"
(minikube) DBG | "SATA-18-0"="none"
(minikube) DBG | "SATA-19-0"="none"
(minikube) DBG | "SATA-20-0"="none"
(minikube) DBG | "SATA-21-0"="none"
(minikube) DBG | "SATA-22-0"="none"
(minikube) DBG | "SATA-23-0"="none"
(minikube) DBG | "SATA-24-0"="none"
(minikube) DBG | "SATA-25-0"="none"
(minikube) DBG | "SATA-26-0"="none"
(minikube) DBG | "SATA-27-0"="none"
(minikube) DBG | "SATA-28-0"="none"
(minikube) DBG | "SATA-29-0"="none"
(minikube) DBG | natnet1="nat"
(minikube) DBG | macaddress1="080027F5725A"
(minikube) DBG | cableconnected1="on"
(minikube) DBG | nic1="nat"
(minikube) DBG | nictype1="82540EM"
(minikube) DBG | nicspeed1="0"
(minikube) DBG | mtu="0"
(minikube) DBG | sockSnd="64"
(minikube) DBG | sockRcv="64"
(minikube) DBG | tcpWndSnd="64"
(minikube) DBG | tcpWndRcv="64"
(minikube) DBG | Forwarding(0)="ssh,tcp,127.0.0.1,1948,,22"
(minikube) DBG | hostonlyadapter2="VirtualBox Host-Only Ethernet Adapter #2"
(minikube) DBG | macaddress2="0800276B6570"
(minikube) DBG | cableconnected2="on"
(minikube) DBG | nic2="hostonly"
(minikube) DBG | nictype2="82540EM"
(minikube) DBG | nicspeed2="0"
(minikube) DBG | nic3="none"
(minikube) DBG | nic4="none"
(minikube) DBG | nic5="none"
(minikube) DBG | nic6="none"
(minikube) DBG | nic7="none"
(minikube) DBG | nic8="none"
(minikube) DBG | hidpointing="ps2mouse"
(minikube) DBG | hidkeyboard="ps2kbd"
(minikube) DBG | uart1="off"
(minikube) DBG | uart2="off"
(minikube) DBG | uart3="off"
(minikube) DBG | uart4="off"
(minikube) DBG | lpt1="off"
(minikube) DBG | lpt2="off"
(minikube) DBG | audio="dsound"
(minikube) DBG | clipboard="disabled"
(minikube) DBG | draganddrop="disabled"
(minikube) DBG | vrde="off"
(minikube) DBG | usb="off"
(minikube) DBG | ehci="off"
(minikube) DBG | xhci="off"
(minikube) DBG | SharedFolderNameMachineMapping1="c/Users"
(minikube) DBG | SharedFolderPathMachineMapping1="\\?\c:\Users"
(minikube) DBG | vcpenabled="off"
(minikube) DBG | vcpscreens=0
(minikube) DBG | vcpfile="C:\Users\asus\.minikube\machines\minikube\minikube\minikube.webm"
(minikube) DBG | vcpwidth=1024
(minikube) DBG | vcpheight=768
(minikube) DBG | vcprate=512
(minikube) DBG | vcpfps=25
(minikube) DBG | GuestMemoryBalloon=0
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | Name:            VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG | GUID:            ce2c60e5-d293-4fbe-9c27-1f6ca0ca9cc6
(minikube) DBG | DHCP:            Disabled
(minikube) DBG | IPAddress:       192.168.99.1
(minikube) DBG | NetworkMask:     255.255.255.0
(minikube) DBG | IPV6Address:     fe80:0000:0000:0000:f40e:d17a:af0f:e7c9
(minikube) DBG | IPV6NetworkMaskPrefixLength: 64
(minikube) DBG | HardwareAddress: 0a:00:27:00:00:20
(minikube) DBG | MediumType:      Ethernet
(minikube) DBG | Status:          Up
(minikube) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG |
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | Searching for hostonly interface for IPv4: 192.168.99.1 and Mask: ffffff00
(minikube) DBG | Found: VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | NetworkName:    HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG | IP:             192.168.99.6
(minikube) DBG | NetworkMask:    255.255.255.0
(minikube) DBG | lowerIPAddress: 192.168.99.100
(minikube) DBG | upperIPAddress: 192.168.99.254
(minikube) DBG | Enabled:        Yes
(minikube) DBG |
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | Removing orphan DHCP servers...
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list hostonlyifs
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | Name:            VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG | GUID:            ce2c60e5-d293-4fbe-9c27-1f6ca0ca9cc6
(minikube) DBG | DHCP:            Disabled
(minikube) DBG | IPAddress:       192.168.99.1
(minikube) DBG | NetworkMask:     255.255.255.0
(minikube) DBG | IPV6Address:     fe80:0000:0000:0000:f40e:d17a:af0f:e7c9
(minikube) DBG | IPV6NetworkMaskPrefixLength: 64
(minikube) DBG | HardwareAddress: 0a:00:27:00:00:20
(minikube) DBG | MediumType:      Ethernet
(minikube) DBG | Status:          Up
(minikube) DBG | VBoxNetworkName: HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG |
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | Adding/Modifying DHCP server "192.168.99.6" with address range "192.168.99.100" - "192.168.99.254"...
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe list dhcpservers
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | NetworkName:    HostInterfaceNetworking-VirtualBox Host-Only Ethernet Adapter #2
(minikube) DBG | IP:             192.168.99.6
(minikube) DBG | NetworkMask:    255.255.255.0
(minikube) DBG | lowerIPAddress: 192.168.99.100
(minikube) DBG | upperIPAddress: 192.168.99.254
(minikube) DBG | Enabled:        Yes
(minikube) DBG |
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm minikube --nic2 hostonly --nictype2 82540EM --nicpromisc2 deny --hostonlyadapter2 VirtualBox Host-Only Ethernet Adapter #2 --c
ableconnected2 on
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm minikube --natpf1 delete ssh
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe modifyvm minikube --natpf1 ssh,tcp,127.0.0.1,1948,,22
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | }
(minikube) DBG | COMMAND: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm minikube --type headless
(minikube) DBG | STDOUT:
(minikube) DBG | {
(minikube) DBG | Waiting for VM "minikube" to power on...
(minikube) DBG | }
E0525 14:27:42.340757   18280 start.go:119] Error starting host: Error starting stopped host: Unable to start the VM: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm minikube --type headless failed:
VBoxManage.exe: error: VT-x is not available (VERR_VMX_NO_VMX)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
Details: 00:00:01.248589 Power up failed (vrc=VERR_VMX_NO_VMX, rc=E_FAIL (0X80004005)).
 Retrying.
(minikube) DBG | STDERR:
(minikube) DBG | {
(minikube) DBG | VBoxManage.exe: error: VT-x is not available (VERR_VMX_NO_VMX)
(minikube) DBG | VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
(minikube) DBG | }
(minikube) DBG | Checking vm logs: C:\Users\asus\.minikube\machines\minikube\minikube\Logs\VBox.log
E0525 14:27:42.353455   18280 start.go:125] Error starting host:  Error starting stopped host: Unable to start the VM: C:\Program Files\Oracle\VirtualBox\VBoxManage.exe startvm minikube --type headless failed:
VBoxManage.exe: error: VT-x is not available (VERR_VMX_NO_VMX)
VBoxManage.exe: error: Details: code E_FAIL (0x80004005), component ConsoleWrap, interface IConsole
Details: 00:00:01.248589 Power up failed (vrc=VERR_VMX_NO_VMX, rc=E_FAIL (0X80004005))

**Comment by gbraad** [comment]
Do you have Hyper-V installed on the same system?

**Comment by reza-ebrahimi** [comment]
@gbraad I have mention it, if that was disabled, cannot open `HyperV Manager`, I have three virtual machines running on `HyperV`.

**Comment by gbraad** [comment]
As mentioned, with:

```
$ cat ~/.minikube/machines/minikube/config.json | grep DriverName
"DriverName": "virtualbox",
```

you show you have a previously defined instance with the virtualbox driver. You need to delete this first, before you can start again. just editing the hupervisor driver in the config is not enough.

**Comment by erikbsap** [comment]
I had to manually delete the minikube directory on windows10 as well to make it work. It would be cool if minikube start had a flag like `--ignore-cache`.

And in any case it is common that cached config is overwritten by written config which is overwritten by shell arguments. This is just the natural way people expect things to work.

**Comment by goudarzi** [comment]
@gbraad I faced the same issue - basically I screwed up my setup by deleting the minikube VM manually. That resulted in facing #459. Then I ran "minikube delete" which resulted in some failures, and tried starting it - for which I ended up here, despite specifying hyperv driver in the command line, receiving the mentioned failures (seemed like it's trying to use VBox instead..).

What worked for me was to delete the .minikube folder (I couldn't find anything VBox related mentioned in config.json file) and then run "minikube start --vm-driver hyperv --hyperv-virtual-switch External".

It's back online now.

**Comment by SebTardif** [comment]
I reached here and got the same original symptom just because I copy-pasted the command from the web and the "-" on the command line was something else, maybe Unicode. I didn't get any warning from minikube that I was passing funny parameters.

**Comment by CharlieReitzel** [comment]
I uninstalled minikube, which ran without complaint (Windows 10, uninstaller).  But it left behind the virtual disk, which was consuming 20GB of disk space.   So I manually deleted the minikube VM (`Remove-VM minikube`) and got the disk space back.  So that's good.  But, now, when I try to delete C:\Users\%USERNAME%\.minikube directory, there is still an orphan snapshot left behind:

{code}
C:\Users\joebob>del /s/q .minikube
C:\Users\joebob\.minikube\machines\minikube\minikube\Snapshots\7D2B9345-2F48-4D9F-98DF-48EC805B7408.vmcx
The process cannot access the file because it is being used by another process.
{code}
